.github/workflows/ci.yaml

```diff
@@ -279,7 +283,7 @@ jobs:
               # still bootstraps Aspect CLI from configuration in .bazeliskrc. Aspect CLI will
               # then use .bazelversion to determine which Bazel version to use
               run: |
-                  echo "${{ matrix.bazelversion }}" > .bazelversion
+                  find . -name ".bazelversion" -type f -exec echo "${{ matrix.bazelversion }}" > {}
```

Comment:
The `.bazelversion` symlinks in bazel-lib 2 e2es that are written back to the source tree are not preserved. Find all non-symlink `.bazelversion` files and replace them.